lib/email_processor.rb

```diff
@@ -32,16 +31,20 @@ def process
       topic = Topic.find(ticket_number)
 
       #insert post to new topic
-      message = "Attachments:" if @email.attachments.present? && @email.body.blank?
+      if attachments_enabled?
+        message = "Attachments:" if @email.attachments.present? && @email.body.blank?
+      end
       post = topic.posts.create(
         :body => message,
         :raw_email => raw,
         :user_id => @user.id,
         :kind => "reply"
       )
 
-      # Push array of attachments and send to Cloudinary
-      handle_attachments(@email, post)
+      if attachments_enabled?
+        # Push array of attachments and send to Cloudinary
+        handle_attachments(@email, post)
```

Comment:
Using a [inline modifier is preferred in ruby](https://github.com/bbatsov/ruby-style-guide#if-as-a-modifier) for single line body. 

`handle_attachments(@email, post) if attachments_enabled?`